module-4/backend/week_two.md

```diff
@@ -5,18 +5,31 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's one difference between ES5 and ES6?
+ES6 uses `const`,  `let`, and `=>`
```

Comment:
Yes! Just make sure you can articulate the difference between the way var and functions with the keyword function behave.